osi_surfaceline.proto

```diff
@@ -0,0 +1,169 @@
+// lb: lane boundary
+//
+// \image html strassenquerschnitt_surfacelines.png "Lane shape surfacelines" width=500px
+// 
+// \attention The points desribing the surface line must be set in the 
+// same ordering (ascending or descending) as the points desribing the 
+// lane boundaries or lane. Example: If the points are deducted from 
+// a map format, the order of points is recommended to be in line with the 
+// road coordinate (e.g. s-coordinate in OpenDRIVE). 
+//
+// \attention The points describing the surface line might be set at
+// arbitrary distances. This allowes to model certain height changes along the 
+// lane like speed bumps etc. more detailed while still allow very few points 
+// at parts of the lane where height does not change much.
+//
+// \note The surface line is the line that describes the height profile 
+// along the lane. Multiple surface lines can be used to create a certain road shape
+// if needed. 
+// Up for discusssion: Should a flat road consist of no surface line, since the 
+// lane boundaries can be used in this case. Other option would be a mandatory surfaceline
+// at each LaneBoundary, in simple flat roads it will consist of just 2 points.
+// 
+// \attention If more than one surface line exists, they are not allowed to cross each other 
+// or the lane boundaries, otherwise you could have conflicting height information at these points.
+//
+message SurfaceLine
+{
+    // The ID of the surface line.
+    // Example: sl1 (see reference picture SurfaceLines).
+    //
+    // \note Note ID is global unique.
+    //
+    // \rules
+    // is_globally_unique
+    // \endrules
+    //
+    optional Identifier id = 1;
+    
+    // points making up the surfaceline
+    // The surfaceline must be defined in the same direction as the 
+    // lane or lane boundaries.
+    //
+    // If the surfaceline approximates a curve (e.g. a cubic function in
+    // OpenDRIVE), the points must be chosen in a way that the lateral distance
+    // to the ideal line does not exceed 5cm. As shown in the following image:
+    //
+    // \image html line_approximation_error.svg "Approximation error"
+    // Approximation error green line.
+    //
+    // The Z error (difference in Z height between surfaceline and the "real"
+    // line) must not exceed 2cm, since height gaps are more noticable.
+    //
+    repeated SurfaceLinePoint surfaceline = 2;
+       
+    // Definition of the placement of the newly introduced surfaceline in the center of the lane.
+    //
+    // Defined and used for driving lanes.
+    // \c true means that the surface line is also the centerline of the lane.
+    //
+    // Example: \c #surfaceline_is_centerline = \c true for surfaceline sl2
+    // and \c #surfaceline_is_centerline = \c false for surfacelane sl1 in
+    // image \ref HighwayExit .
+    //
+    // \note The \c #centerline is defined only for \c #type =
+    // \c #TYPE_DRIVING and if exactly one or no \c #lane_pairing pair
```

Comment:
Failing build test: Linking fields from other files/messages need some reference to the concerning message. Probably Lane::Classification::type, Lane::Classification::TYPE_DRIVING and Lane::Classification::lane_pairing?

Reply:
I added the appropriate references